### ISSUE TITLE
Should we break the collector app into its own repo?

### ISSUE THREAD

**Issue by groovecoder** [mdn/webalyzer#16]
The collector app makes a good stand-alone system/service. If we put it into a dedicate repo, we could run it independently from analyzer code.

**Comment by peterbe** [comment]
I agree but I don't agree that it should be its own repo. At the moment, the app is completely independent from the analyzer (which technically should be renamed to `redundantcss` or something). 

But I like that it's under one set of settings and that any of the plugins can just get to the collected html with `from webalyzer.collector.models import Page`

**Comment by groovecoder** [comment]
As [I try](https://github.com/groovecoder/webalyzer/tree/heroku-11) to [get this running on Heroku](https://github.com/mdn/webalyzer/issues/11), it seems like a mis-match to have python/django collector, python analyzer (mincss), and node analyzer (crass) all mashed up in a single repo/system.

And the way the python code uses subprocess to run the node analyzers or tasks creates some tricky-to-debug issues. E.g., when [the gator worker process tries to use a phantomjs process to generate a screenshot in the collected app.](https://github.com/mdn/webalyzer/issues/13)
